src/logic/consts.js

```diff
@@ -0,0 +1,18 @@
+export const gameField = 'gameField';
```

Comment:
what are all these constants about? I see some of them are related to actions, some - not. you should follow 1 principle: 1 module - 1 responsibility